static/js/util/profile_edit.js

```diff
@@ -220,6 +221,57 @@ export function boundCheckbox(keySet: string[], label: string|React$Element<*>):
   );
 }
 
+const onTelChange = R.curry((
+  keySet,
+  countryKeySet,
+  profile,
+  updateProfile,
+  validator,
+  newPhoneNumber,
+  countryObj
+) => {
+  let currentPhoneNumber = _.get(profile, keySet);
+  if ( currentPhoneNumber !== newPhoneNumber ) {
+    let clone = _.cloneDeep(profile);
+    _.set(clone, keySet, newPhoneNumber);
+    _.set(clone, countryKeySet, countryObj.iso2);
```

Comment:
@aliceriot sorry for some reason I did not see this comment. These data fields in the profile are for sending to Pearson, so that's why they're structured that way (see https://github.com/mitodl/micromasters/blob/master/exams/pearson_test.py#L79). For example, if your full US phone number is `1-978-123-4567`, then the code we already have in place to sync this data to Pearson will expect `phone_number = '978-123-4567'` and `phone_country_code = '1'`. Otherwise, if you think sorting them in an alternative way is better, we can discuss that, but the corresponding Pearson profile sync code will need to be adjusted accordingly.